src/init.cpp

```diff
@@ -2378,7 +2378,7 @@ bool AppInitMain(NodeContext& node, interfaces::BlockAndHeaderTipInfo* tip_info)
 
     const auto BadPortWarning = [](const char* prefix, uint16_t port) {
         return strprintf(_("%s request to listen on port %u. This port is considered \"bad\" and "
-                           "thus it is unlikely that any Dash Core peers connect to it. See "
+                           "thus it is unlikely that any peer will connect to it. See "
```

Comment:
6396df0056ef34b23097f6a10055b34d6e22ff91 :
> Dash Note: Only applied init.cpp change as Dash doesn't have external signer UI elements.

*yet*. Should be `partial` imo.

Reply:
bot; you do not need to make changes based on this feedback